### DISCUSSION TITLE
How do you compress templates in flask?

### DISCUSSION THREAD

**Discussion by zhenzi0322** [pallets/flask#4745] [Q&A]
How to compress the template into one line

**Comment by ThiefMaster** [comment]
usually you don't

**Reply by zhenzi0322** [reply]
> usually you don't

That's why I ask, how to compress the template?

**Reply by ThiefMaster** [reply]
I guess you talk about compressing the resulting HTML before sending it to the client, right? Because compressing the actual templates would be pointless.

But even compressing the HTML in such a way is not very useful: gzipping responses is a thing (and your web server between Flask and the end user can usually do that), an HTML can be compressed very efficiently. Most likely more efficiently than you stripping unnecessary whitespace will ever be.
